package.json

```diff
@@ -28,7 +28,7 @@
     "i18n-js": "^3.3.0",
     "react": "16.8.3",
     "react-dom": "^16.8.6",
-    "react-native": "0.59.10",
```

Comment:
This will break our compilation

Reply:
Yes, it is outdated now as well, are you able to modify that file apart from the commit i recently did?, so that it passes the tests